### ISSUE TITLE
losetup -f returns free loopback device

### ISSUE THREAD

**Issue by lordfolken** [hor63/horOpenVario#6]
I always had collision with snapd. So a possiblity is that losetup -f gives you a free loopback device. 

https://github.com/hor63/horOpenVario/blob/3c50869d9dc52294246b261432be218d48821200/makenewimage.sh#L158

```bash
# losetup --help
-f, --find                    find first unused device
```
```bash
LOOPBACKDEV=$(losetup -f)

**Comment by hor63** [comment]
Good Idea. Thanks. Will check in the next days on vacation.

**Comment by hor63** [comment]
Implemented with commit [1059e6b](https://github.com/hor63/horOpenVario/commit/1059e6b377348fdddd9d46670a96c407fd1af03e)
